CONTRIBUTING.md

```diff
@@ -39,35 +39,95 @@ Check the "Developer Setup" below for the developer workflow.
 
 ## 2. 🔨 Developer Setup
 
-First install [anaconda](https://docs.anaconda.com/free/anaconda/install/windows/) and add to path by going to advanced system settings. Then, launch cmd.
+First install [anaconda](https://docs.anaconda.com/free/anaconda/install/windows/) and follow the recommended settings. Once installed, open up the "Anaconda Prompt" and type in the command "conda --version". If everything has worked, it should print "conda xx.xx.x".
+For example: "conda 23.11.0
+
+(P.S: You may add Anaconda to your path environment variables on your own (if you feel comfortable), however this is discouraged by Anaconda themselves as it may "interfere with other software". Instead, they recommend to use the Anaconda Prompt.)
 
 Ensure you have `make`. if it's not installed on your computer, Please follow this [article](https://earthly.dev/blog/makefiles-on-windows/).
 
 
-Use the following command to create a conda environment `discussion-agents` with Python version 3.10.13. Any Python version above 3.9 is viable.
+Open up the Anaconda Command Prompt and use the following command to create a conda environment `discussion-agents` with Python version 3.10.13. Any Python version above 3.9 is viable.
+
 ```
 conda create -n discussion-agents python=3.10.13
 ```
+
 Now activate the environment.
+
 ```
 conda activate discussion-agents
 ```
+
```

Comment:
doesn't need to be in anaconda prompt (only applicable to windows). just needs to be where conda is supported